test_app/tests/tests_model.py

```diff
@@ -476,3 +476,20 @@ def test_create_before_range(self):
         self.assertOrder(placement_2, 2)
         self.assertOrder(placement_3, 3)
         self.assertOrder(placement_4, 0)
+
+    def test_delete_within_range(self):
+        year = 1900
+
+        placement_1 = TriWizardPlacement.objects.create(year=year, wizard=self.wizards[0], prize="rock")
+        placement_2 = TriWizardPlacement.objects.create(year=year, wizard=self.wizards[1], prize="stone")
+        placement_3 = TriWizardPlacement.objects.create(year=year, wizard=self.wizards[2], prize="wand")
+        placement_4 = TriWizardPlacement.objects.create(year=year, wizard=self.wizards[3], prize="hug")
+
+        placement_2.delete()
```

Comment:
Will `TriWizardPlacement.objects.delete(pk= placement_2.pk)` work too?
I **think** the queryset is called more often than the actual `.delete`.

Reply:
I added a post_delete signal to address that